### ISSUE TITLE
Player `spatialVolume` and `nonSpatialVolume` setters do not change anything

### ISSUE THREAD

**Issue by xxshady** [altmp/altv-issues#2183]
### Description of the problem

/

### Reproduction steps

Use setters on remote players clientside, maybe affected by `enableSyncedMetaData = false` in server config

```js
const [, player] = alt.Player.all;
player.spatialVolume = 0.1;
player.nonSpatialVolume = 0.1;
```

### Expected behaviour

/

### Operating system

Windows 11

### Version

16.0.0-dev.126

### Confirmation of issue's presence

- [X] *By submitting this ticket, I affirm that I have verified the presence of this issue on the latest developer version available at the time of writing this ticket.*

**Comment by litolax** [comment]
Confirmed

**Comment by xxshady** [comment]
Fixed in 16.0.0-dev.149
